specification/subscription/resource-manager/Microsoft.Subscription/preview/2020-01-01-preview/subscriptions.json

```diff
@@ -0,0 +1,469 @@
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Subscription/OperationResults/PurchaseSupport_{subscriptionId}:{purchaseId}": {
+      "get": {
+        "operationId": "GetPurchaseSupportStatus_Get",
+        "x-ms-examples": {
+          "PurchaseOperationSupportStatus": {
+            "$ref": "./examples/PurchaseOperationSupportStatus.json"
+          }
+        },
+        "description": "The operation to find out the purchase Operation status",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/purchaseIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/SupportPlan"
+            }
+          },
+          "202": {
+            "description": "Accepted. Purchase support plan is in progress.",
+            "headers": {
+              "Location": {
+                "description": "GET this URL to retrieve the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/SupportPlan": {
+      "get": {
+        "operationId": "SupportPlans_list",
+        "x-ms-examples": {
+          "SupportPlan": {
+            "$ref": "./examples/SupportPlanList.json"
+          }
+        },
+        "description": "The operation to find out support plan list",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/SupportPlanResponseResult"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
```

Comment:
>} [](start = 8, length = 1)

Add support for paging..  you m ay chose to return null.. but don't  need to rev api version just to update this
"x-ms-pageable": {
          "nextLinkName": "nextLink"
        }

Reply:
done